doc/man/8/radosgw-admin.rst

```diff
@@ -805,7 +805,7 @@ Options
 
 .. option:: --caps=<caps>
 
-	List of caps (e.g., "usage=read, write; user=read".
+	List of caps (e.g., "usage=read, write; user=read").
```

Comment:
please use lower case "doc" in the title, and better off being more specific, like

> doc/man: add missing right parenthesis in manpage.